### ISSUE TITLE
(Bug) wallet balance not updating when switching accounts

### ISSUE THREAD

**Issue by sirpy** [GoodDollar/GoodProtocolUI#99]
when switching account on metamask the balances do not update
![image.png](https://images.zenhubusercontent.com/5d5081131325bafe066d2c4e/7944b94f-3255-4489-b109-f028ce957b6d)

**Comment by L03TJ3** [comment]
Is it true this has been fixed already?

**Comment by L03TJ3** [comment]
@sirpy 

So, I have now tested this across the board. Both fi-qa and goodSwap.
And locally from develop > production.
For me, it works whenever I switch accounts/switch networks. It shows the amount of G$/GDX/GOOD I have or don't have.
also in code, it seems to get all the right token contracts per network/env.

Was there a specific env you were on? switching to a specific network?

Small note is that I have not been able to test all networks with an amount on my address, as I was not able to find a way to get G$'s for all. I am sure about Fuse / Eth Main / Kovan.
For the separate fuse contracts develop/staging/ I didn't know how to get tokens for them..
But at least the right contract is loaded, that I do know!

Might just be not having G$ for that specific contract?
That's all I got for now...
